www/src/pages/about.mdx

```diff
@@ -0,0 +1,28 @@
+# About
+<p className="lead">
+	Get to know more about the team maintaining React-Bootstrap. Learn a 
+	little history of how, why and when the project started and how
+	you can be a part of it.
+</p>
+
+### Team
+React-Bootstrap is maintained by a [team of developers](https://github.com/orgs/react-bootstrap/people) on Github. We have a growing team
+and if you are interested in re-building the most popular front-end framework with React we would love to hear from you.
+
+### Contributors
+We welcome community support with both pull requests and reporting bugs. Please don't hesitate to jump in.
+Join our growing list of [contributors](https://github.com/react-bootstrap/react-bootstrap/graphs/contributors).
+
+### Sponsorships
+We currently aren't accepting any funding what so ever, but that could change 😉
```

Comment:
I'm not sure if we should even have this blurb? I haven't really discussed with the other maintainers as to their thoughts on funding and sponsorships yet, we could just omit this section for now.

Reply:
Done